modules/runner/src/main/java/org/apache/ignite/internal/configuration/CoreDistributedConfigurationModule.java

```diff
@@ -52,7 +63,18 @@ public ConfigurationType type() {
         return List.of(
                 HashIndexConfigurationSchema.class,
                 SortedIndexConfigurationSchema.class,
-                PartialIndexConfigurationSchema.class
+                PartialIndexConfigurationSchema.class,
+                UnknownDataStorageConfigurationSchema.class
+        );
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public Map<Class<? extends Annotation>, Set<Validator<? extends Annotation, ?>>> validators() {
+        return Map.of(
+                KnownDataStorage.class, Set.of(KnownDataStorageValidator.INSTANCE),
+                TableValidator.class, Set.of(TableValidatorImpl.INSTANCE),
```

Comment:
Wait a minute, why did you add TableValidatorImpl? Isn't it supposed to be in another module already?

Reply:
My mistake, didn't notice the duplicate.